schemas/session-replay/mobile/touch-data-schema.json

```diff
@@ -1,55 +1,50 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "$id": "touch-data-schema.json",
-    "title": "TouchData",
-    "type": "object",
-    "description": "Schema of a TouchData.",
-    "allOf": [
-        {
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "touch-data-schema.json",
+  "title": "TouchData",
+  "type": "object",
+  "description": "Schema of a TouchData.",
+  "allOf": [
+    {
+      "properties": {
+        "source": {
+          "type": "integer",
+          "const": "2",
```

Comment:
here should be 2 instead of "2", was already corrected on master, a rebase will solve it.

Reply:
Fixed